tests/ui/test_ui.py

```diff
@@ -309,17 +309,19 @@ def test_keypress_autohide_streams(self, view, mocker, autohide, key,
          .assert_called_once_with(view.stream_w.stream_search_box))
 
     @pytest.mark.parametrize('key', keys_for_command('SEARCH_PEOPLE'))
-    def test_keypress_edit_mode(self, view, mocker, key):
+    def test_keypress_edit_mode(self, view, mocker, key, widget_size):
         view.users_view = mocker.Mock()
         view.body = mocker.Mock()
         view.user_search = mocker.Mock()
 
         super_view = mocker.patch("zulipterminal.ui.urwid.WidgetWrap.keypress")
 
         view.controller.is_in_editor_mode = lambda: True
-        size = (130, 28)
+        size = widget_size(view.main_window)
```

Comment:
This commit references `view.main_window` but tests `view`, too.

Reply:
(Posted an explanation in the other comment.)